venues/MIDL.io/2019/Conference/python/post-submission-stage-1.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python
+
+import sys, os
+import argparse
+import datetime
+import openreview
+import config
+
+"""
+OPTIONAL SCRIPT ARGUMENTS
+	baseurl -  the URL of the OpenReview server to connect to (live site: https://openreview.net)
+ 	username - the email address of the logging in user
+	password - the user's password
+"""
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print('connecting to {0}'.format(client.baseurl))
+
+conference = config.get_conference(client)
+
+conference.close_submissions(freeze_submissions = True)
+
```

Comment:
You should also add the conference to active_venues if it's not already there.

Reply:
it is in this PR: https://github.com/iesl/openreview-py/pull/198